app/actions.js

```diff
@@ -166,3 +167,17 @@ export const getTokenList = () => ({
 		}
 	}
 });
+
+export const sendToken = (tokenObject) => ({
+	type: SEND_TOKEN,
+	meta: {
+		offline: {
+			effect: {
+				url: `${API_URL}/tokens/insert`,
+				method: 'POST',
+				body: JSON.stringify(tokenObject)
+			},
+//			commit: { type: LOGIN },
```

Comment:
Is this supposed to be left here?  Do we want to have an action that fires if it succeeds? I recall there being a `getTokenList`.  Do we need that anymore or can we just return the token from this new action?  Not too sure what the getTokenList is for, so feel free to correct me haha

Reply:
We need getTokenList before we send the token, because we don't want to send the token if it's already in the list. So I don't think we have any action to fire if sending the token succeeds? I can take out the comment.